<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashDialog</class>
 <widget class="QDialog" name="SplashDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>241</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>T.C.L Turbine Counter &amp; Logger</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Available Ports:</string>
   </property>
   <widget class="QComboBox" name="mAvailablePorts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="mTitleBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QLabel" name="mTitleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="mMainImage">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/windmill.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="mConnectButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&amp;Connect</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mRefreshButton">
      <property name="text">
       <string>&amp;Refresh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mExitButton">
      <property name="text">
       <string>&amp;Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
